app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -0,0 +1,144 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    # Generic JWT authenticator that receive JWT vendor configuration and uses to validate that the authentication
+    # request is valid, and return conjur authn token accordingly
+    Authenticator = CommandClass.new(
```

Comment:
i see that you no longer have the regular class with the `valid?` function. this means that the `authn-jwt` authenticator will not be present in the `/authenticators` and `/info` endpoints. 

is that your intent? 

cc: @nessiLahav

Reply:
@orenbm Nice catch , i opened a bug about this and we will discuss with PO if it for this version